src/data/taskList.ts

```diff
@@ -1,4 +1,4 @@
-import { SystemIntakeForm } from 'types/systemIntake';
+import { GetSystemIntake_systemIntake as SystemIntake } from 'queries/types/GetSystemIntake';
```

Comment:
It's great to be able to leverage the GraphQL types more instead of having to define them all ourselves.

I like this use of `as` to make the local name for the type more succinct.